WinForms/PropertyEditing/PropertyGrid.cs

```diff
@@ -88,6 +89,18 @@ public void ApplyTo(PropertyEditor mainEditor, bool dontCollapse = true)
 				child.Expanded = this.IsEditorExpanded(child);
 			}
 		}
+		public void SaveToXml(XElement node)
+		{
+			string expandedString = string.Concat(this.expandedNodes.Select(s => s + ","));
+			node.SetElementValue("ExpandedNodes", expandedString);
```

Comment:
A comma-separated list is a very un-XML way of expressing a list of values. Can we do an element list instead? Something like:

```xml
<ExpendedNodes>
  <NodeId>Foo</NodeId>
  <NodeId>Bar</NodeId>
  ...
</ExpandedNodes>
```